tools/e2etesting/steps/deployplatform.yml

```diff
@@ -26,6 +26,15 @@ steps:
     SecretsFilter: 'ContainerRegistryPassword,ContainerRegistryServer,ContainerRegistryUsername'
     RunAsPreJob: true
 
+- task: AzureKeyVault@1
+  displayName: 'Load secrets from Release Key Vault as variables on Release Branch'
+  condition: eq(variables['BranchName'], 'release')
```

Comment:
I would say this is not sufficient, e.g. when release branch is is hierachical (/release/Xyz), we need something like StartsWidth or in variables.yml some cut at the end.

Reply:
per discussion, we will need this for release/lts and also for older releases like release/2.7.206.